tox.ini

```diff
@@ -51,10 +51,11 @@ basepython = python3
 skip_install = true
 deps =
     flake8
+    flake8-rst-docstrings
     git+https://github.com/aws/amazon-braket-build-tools.git
 commands =
     flake8 {posargs}
-    flake8 --enable-extensions=BCS src
+    ; flake8 --enable-extensions=BCS src
```

Comment:
Any reason to comment this out instead of removing it entirely?

Reply:
Fixed